SSA/Projects/InstCombine/scripts/cfg.py

```diff
@@ -8,7 +8,7 @@
 from enum import Enum, auto
 
 # The path from lean-mlir to llvm-project
-llvm_path = "../llvm-project-main"
+llvm_path = "../llvm-project"
 
```

Comment:
```suggestion
llvm_path = "../llvm-project-main"
```

This change seems unrelated.